f/frizzle/comapeo/comapeo_observations.py

```diff
@@ -65,17 +83,15 @@ def fetch_comapeo_projects(
     logger.info("Fetching projects from CoMapeo API...")
     response = requests.request("GET", url, headers=headers, data=payload)
 
-    if response.status_code == 404:
-        logger.error(f"URL not found: {url}")
+    if response.status_code != 200:
+        logger.error(f"Unexpected status code: {response.status_code}")
         raise ValueError(
             f"Failed to fetch projects: {response.status_code} - {response.reason}"
         )
```

Comment:
```suggestion
    response.raise_for_status()
```


[FYI this method exists.](https://3.python-requests.org/user/quickstart/#response-status-codes)

Don't accept the suggestion if you think there's good value in this extra formatting of the error that you add.